libs/domain/cart/edit/edit.component.ts

```diff
@@ -111,30 +110,38 @@ export class CartEditComponent extends CartComponentMixin(
     const fields = this.getFields();
     const values = this.$values();
 
-    return html`<form @submit=${this.onSubmit}>
+    return html`<form @oryx.submit=${this.onSubmit}>
```

Comment:
looks like it's BC

Reply:
This component added as a part of this feature, have not yet released